examples/upsert-resource-multiple/.expected/config.yaml

```diff
@@ -0,0 +1,5 @@
+testType: eval
+image: upsert-resource:unstable
+fnConfig: fn-config.yaml
+# Skip this until v1.0.0-beta.9 of kpt is released
```

Comment:
This test can be unskiped as soon as we merge https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/644/commits.